docker/api/Dockerfile

```
@@ -21,11 +21,16 @@ RUN git clone https://github.com/SUNET/cnaas-nms.git
 
 WORKDIR /opt/cnaas/venv/cnaas-nms/
 
-RUN python3 -m pip install -r requirements.txt
+RUN pip3 install -r requirements.txt
 COPY db_config.yml /etc/cnaas-nms/db_config.yml
 COPY api.yml /etc/cnaas-nms/api.yml
 COPY repository.yml /etc/cnaas-nms/repository.yml
+COPY entrypoint.sh /usr/local/bin/
 
 WORKDIR /opt/cnaas/venv/cnaas-nms/src/
 
-ENTRYPOINT PYTHONPATH=`pwd` python3 cnaas_nms/run.py
+EXPOSE 5001
```

I think this should be port 5000?
```suggestion
EXPOSE 5000
```

---

Fixed.